src/app/cube/home/help/teach-now/teach-now.component.html

```diff
@@ -1,11 +1,21 @@
 <div class="wrapper">
-    <div #topicScroll *ngIf="width > 750" class="topics">
-        <div class="shadow left" aria-hidden="true"></div>
-        <!-- NOTE: Add left and right buttons for scrolling if it becomes an issue -->
-        <button *ngFor="let topic of topics" class="topic-btn" (activate)="selectTopic(topic._id)" [ngClass]="{'selected': selectedTopic === topic._id}">{{ topic.name }}</button>
-        <span *ngIf="topics.length === 0">Loading Topics...</span>
-        <div class="shadow right" aria-hidden="true"></div>
+
+    <div *ngIf="width > 750" class="topics">
+        <button (activate)="horizontalScrollOnClick(false)">
```

Comment:
Let's remove the default styles on these buttons to match the other chevron's we have with no background and one of the clark greys as it's color

Reply:
I made the buttons the same as the topics buttons. Let me know if that doesn't work

Reply:
This works! @pzalep1 any thoghts?